packages/core/src/card/Card.css

```diff
@@ -1,9 +1,11 @@
 /* Styles applied to the root element */
 .saltCard {
   box-shadow: var(--saltCard-boxShadow, var(--salt-overlayable-shadow));
-  border-width: var(--salt-size-border);
+  border-width: var(--saltCard-borderWidth, var(--salt-size-border));
   border-style: var(--salt-container-borderStyle);
   padding: var(--saltCard-padding, var(--salt-spacing-300));
+  position: relative;
+  --card-accentColor: var(--saltCard-accentColor, var(--salt-selectable-foreground));
```

Comment:
nit: I would put this as --card-accent-color as the accent and color parts are 2 different semantics
same for the api variable :)

Reply:
Thanks, I think I had this before, but it was failing lint:style "salt/custom-property-attributes-kebab-case".

I'm not sure how it's not kebab case though..